File: materials/5_js/tasks/easy/08.js
/**
 * Количество гласных в строке
 * Ввод: строка
 * Вывод: количество гласных букв (`a`, `e`, `i`, `o`, `u`)
 * Например, pneumonoultramicroscopicsilicovolcanoconiosis - 20 гласных
 */
const string = prompt("Введите строку");
const glasnye = ["a", "e", "i", "o", "u"];
let count = 0;
for (letter of string) {
  for (let i = 0; i < glasnye.length; i++) {
    if ((letter === glasnye[i])) {
      count += 1;
      break;
    }
  }
}
alert(count);

File: frontend/other/0_python/tasks/easy/08.py
'''
Количество гласных в строке
Ввод: строка
Вывод: количество гласных букв (`a`, `e`, `i`, `o`, `u`)
Например, pneumonoultramicroscopicsilicovolcanoconiosis - 20 гласных
'''
s = input("Введите строку: ")
vowels = {'a', 'e', 'i', 'o', 'u'}
count = 0
for char in s.lower():
  if char in vowels:
    count += 1
print(count)
